common/sgx/qeidentity.c

```diff
@@ -32,84 +32,97 @@ void dump_info(const char* title, const uint8_t* data, const uint8_t count)
     }
 }
 
-oe_result_t oe_enforce_qe_identity(sgx_report_body_t* qe_report_body)
+oe_result_t oe_validate_qe_report_body(sgx_report_body_t* qe_report_body)
```

Comment:
This function should be removed it is not used anywhere.

Reply:
It is used in oe_verify_quote_internal().